docs/development-guide.md

```diff
@@ -1,30 +1,22 @@
 # Development and cookiecutter installation
 
-1. [Getting started](getting-started.md)
-2. [Development and installation](development-guide.md)
-3. [Deployment for production](deployment-guide.md)
-4. [Authentication and magic tokens](authentication-guide.md)
-5. [Websockets for interactive communication](websocket-guide.md)
+This section will walk through running the cookiecutter and setting the necessary configuration variables to have a smooth development process. **It is highly advised you read through the input variables before running the cookiecutter command**
```

Comment:
Can change to just "This section will walk through setting up your local environment for developing a generated app."